solution-base/mongodb/Makefile

```diff
@@ -0,0 +1,23 @@
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+CHART_DIR:="${ROOT_DIR}/charts"
+
+CHART_REPO:="https://charts.bitnami.com/bitnami"
+CHART_VERSION:="7.8.0"
+
+PATCH_DIR:="${ROOT_DIR}/patches"
+PATCH_FILES:="$(shell ls -d ${PATCH_DIR}/*)"
+
+HELM=helm
```

Comment:
Curious what the reason for this is? Just in anticipation of changing later on?

Reply:
Typically it would be `?=` to allow temporarily using another `helm` binary (with `make ... HELM=/my/other/helm`) than the default one in the path, to quickly test something or work around a bug without changing the makefile.

Reply:
gotcha, ok. That is more or less what I was thinking.